src/components/PhotoAdd/PhotoAdd.js

```diff
@@ -0,0 +1,45 @@
+import React, {useState} from 'react';
+import './PhotoAdd.css'
+
+const PhotoAdd = ({updateProfilePicture, currentProfilePic}) => {
+    const [photoFilePath, setPhotoFilePath] = useState(currentProfilePic)
+    const [previewHeader, setPreviewHeader] = useState("Current Profile Picture")
+
+    const handleSubmit = (e) => {
+        setPreviewHeader("Current Profile Picture")
```

Comment:
We may have to add a parameter to this function to make this component reusable.

Reply:
Yep we will have to change quite a few things with the new condition